Add ModernAuth (#309)
MicrosoftDocs/OfficeDocs-SharePoint-PowerShell: sharepoint/sharepoint-ps/sharepoint-online/Connect-SPOService.md

Connects to a SharePoint Online Administration Center specifying the region.

### -----------------------EXAMPLE 5-----------------------------

 ```powershell

> Connect-SPOService -Credential $creds -Url https://tenant-admin.sharepoint.com -ModernAuth $true -AuthenticationUrl https://login.microsoftonline.com/organizations

Reviewer: Please enclose the PS code with a triple-backtick:
https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide?view=powershell-7.2#markdown-for-code-samples

Location for AAD Cross-Tenant Authentication service. Can be optionally used if non-default Cross-Tenant Authentication Service is used.

### -----------------------EXAMPLE 5-----------------------------

>  ```powershell
> Connect-SPOService -Credential $creds -Url https://tenant-admin.sharepoint.com -ModernAuth $true -AuthenticationUrl https://login.microsoftonline.com/organizations

Reviewer 2: ```suggestion
 ```powershell
Connect-SPOService -Credential $creds -Url https://tenant-admin.sharepoint.com -ModernAuth $true -AuthenticationUrl https://login.microsoftonline.com/organizations
 ```
```

Accept wildcard characters: False
```

### -ModernAuth

>  Ensure SPO tenant administration cmdlets can connect to the service using modern TLS protocols.

Reviewer 2: ```suggestion
 Ensures that SharePoint Online tenant administration cmdlets can connect to the service using modern TLS protocols.
```

> To use it you also need to provide the AuthenticationUrl parameter.

Reviewer 2: ```suggestion
To use it you also need to provide the **AuthenticationUrl** parameter.
```

Example:

> $creds = Get-Credential
> Connect-SPOService -Credential $creds -Url https://tenant-admin.sharepoint.com -ModernAuth $true -AuthenticationUrl https://login.microsoftonline.com/organizations

Reviewer 2: ```suggestion

$creds = Get-Credential

```powershell
Connect-SPOService -Credential $creds -Url https://tenant-admin.sharepoint.com -ModernAuth $true -AuthenticationUrl https://login.microsoftonline.com/organizations
```

```

Accept wildcard characters: False
```

### -ModernAuth

 Ensures that SharePoint Online tenant administration cmdlets can connect to the service using modern TLS protocols.

To use it you also need to provide the **AuthenticationUrl** parameter.

Example:

$creds = Get-Credential

 ```powershell

> Connect-SPOService -Credential $creds -Url https://tenant-admin.sharepoint.com -ModernAuth $true -AuthenticationUrl https://login.microsoftonline.com/organizations

Reviewer 3: I tested this example and it only worked if I removed the -Credential, can you please check? Thanks

Author: Confirmed on multiple machines this works.

Did you ever get a prompt when you ran $creds = Get-Credential? If not , you may need to run PowerShell in administrator mode.

Reviewer 3: Yes, I got a basic auth prompt, with a non-MFA account it worked, but with a MFA enabled account it failed, maybe I am supposed to do something different? Thanks for clarifying

Author: Ya sorry for MFA it would be minus credential. If we need to move this section to be clearer, that's fine.

Reviewer 3: Yes, can you do it? We do not have permissions to edit the PR, thanks
